Debian Bug report logs - #479803
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=479803

liblocale-gettext-perl: defines LC_* slightly differently from Perl 5.10's
POSIX.pm

New patch debian/patches/compatibility-with-POSIX-module.diff so that
Locale::Gettext re-exports the LC_* constants coming from POSIX instead
of those coming from its own XS implementation.

--- gettext-1.05/gettext.pm.orig	2015-05-14 09:06:58.288075712 -0700
+++ gettext-1.05/gettext.pm	2015-05-14 09:07:23.772010237 -0700
@@ -32,6 +32,7 @@
 =cut
 
 use Carp;
+use POSIX qw(:locale_h);
 
 require Exporter;
 require DynaLoader;
